libtenzir/src/tql2/eval_binary.cpp

```diff
@@ -27,25 +27,34 @@ namespace tenzir {
 namespace {
 
 [[maybe_unused]] constexpr auto is_arithmetic(ast::binary_op op) -> bool {
+  using enum ast::binary_op;
   switch (op) {
-    using enum ast::binary_op;
     case add:
     case sub:
     case mul:
     case div:
```

Comment:
Is there a case for `mod` as well? I could imagine that it's useful for load-balancing.